src/apis/computed.ts

```diff
@@ -9,7 +9,7 @@ interface Option<T> {
 }
 
 // read-only
-export function computed<T>(getter: Option<T>['get']): Readonly<Ref<T>>;
+export function computed<T>(getter: Option<T>['get']): Readonly<Ref<Readonly<T>>>;
```

Comment:
Why we need to type T as Readonly.

Reply:
My suggestion might not be correct, based on the rfc, it returns an immutable ref object, I think you shouldn't be allowed to change it, but 2.x allows you to do that.

I actually open an rfc to discuss that: https://github.com/vuejs/rfcs/pull/82

I apologise if I jump into conclusions to early, the [documentation for computed](https://vue-composition-api-rfc.netlify.com/api.html#computed) also changed because of me :/

Reply:
Should I to do anything?